include/buildir.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
 # duplicated in s/include/common.sh, needed for bootstrapping:
-. include/common.sh
+. $ROOT_DIR/include/common.sh
```

Comment:
how does this work? isn't $ROOT_DIR undefined until one sources include/common.sh?

Reply:
It's in the top level of the Makefile, so it's always present :)